### ISSUE TITLE
Possible Screen reader enhancements

### ISSUE THREAD

**Issue by jfmherokiller** [Nuku/Flexible-Survival#2396]
https://i7el.herokuapp.com/extensions/simple-spelling-by-alice-grove seems to provide a method of "spelling out" items that may help with screen readers. I have tested the extension myself (sadly not with a screen reader but it does mix with the codebase well). The biggest issue with using the above extension is that the initial question doesn't have correctly working hyperlinks because it appears before the hyperlinks are turned on.
